controls/radrichtextbox/document-elements/features-section.md

```diff
@@ -32,7 +32,7 @@ As explained in the previous section, the state of the document is essential for
 

```

Comment:
I think this refers to a section in another topic and the sentence doesn't make much sense here.